docs/components/modeler/bpmn/message-events/message-events.md

```diff
@@ -8,6 +8,12 @@ Message events are events which reference a message; they are used to wait until
 
 ![process](assets/message-events.png)
 
+Message events are used if a single process instance needs to wait for a message from a secondary process or an external system. This is typically a single sender to a single recipient relationship (1:1).
+
+These differ from [signal events](/components/modeler/bpmn/signal-events/signal-events.md), which are used if you want to communicate with (potentially) multiple listeners. For intermediate events, a signal will trigger all process instances with a token waiting at a corresponding catch event, even across different processes. For start events, a signal will start one instance per process that has a corresponding signal start. Thus, signals form a single sender to several-recipient relationship.
```

Comment:
I could still see trouble differentiating between "typically" and "potentially" used here. To me, this implies a message event could sometimes have multiple recipients, and a signal event could sometimes have one - so in those instances, it's less clear how they'd be used differently. Is this the intended difference, or does another situation apply?

Reply:
We could make it clear that a Message can have at most one recipient, while signal can trigger any number of events (0..N).

Reply:
Adjusted some of the wording in my latest commit 👍